src/tests/unit/pages/profile-page/avatar/avatar.spec.js

```diff
@@ -44,25 +44,26 @@ describe('Profile=Page test image restrictions', () => {
     expect(screen.getByText('error.profile.extension')).toBeInTheDocument();
   });
 
-  it('should give error for wrong dimension of fake file', async () => {
-    const file = new File(['test file'], 'fake-small-image.png', { type: 'image/png' });
+  it('should upload image', async () => {
+    const file = new File(['test file'], 'image.jpeg', { type: 'image/png' });
 
     fireEvent.change(uploader, { target: { files: [file] } });
 
-    expect(await screen.findByText('error.profile.dimension')).toBeInTheDocument();
+    expect(uploader.type).toBe('file');
+
+    expect(uploader.files.length).toBe(1);
   });
+  // it('should show preview image', async () => {
```

Comment:
Add explanation to a commented piece of code, or remove it.

Reply:
tests have been refactored, comments deleted